examples/2016-10-31/api_resource_policy/template.yaml

```diff
@@ -0,0 +1,28 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Globals:
+  Api:
+    Auth:
+      ResourcePolicy:
+        CustomStatements: [{
```

Comment:
What is the use case for adding this custom statement? For tests, the validity of the custom statement isn't as important, just that we transform it correctly. However, for examples, I think we should show a valid use case for custom statements. I'm not sure this is one.

Reply:
I changed it to allowing access based on the IP address.